test/integration/org_admin_js_test.rb

```diff
@@ -95,6 +95,7 @@ def test_org_admin_tries_to_create_domain_when_unselect_the_organization
       ensure_selected_option_of_multiselect(@org1.name, select_id: 'domain_organization_ids')
       page.click_button 'Submit'
 
+      wait_for { find('.pf-v5-c-page__main-breadcrumb h1').visible? rescue false }
```

Comment:
Since `find` by default waits and only shows visible elements I'd expect this to also work:
```suggestion
      find('.pf-v5-c-page__main-breadcrumb h1')
```

Though I wonder if we can assert a domain with `domain.name` exists on the page we're redirected to. That would be a good UX anyway IMHO. Not blocking on that here, but perhaps that's really something we should pick up as we're rewriting existing pages to PF5.

Stolen from the test above:
```suggestion
      within "#domains_list" do
        assert page.has_link?(domain.name)
      done
```

Reply:
it doesnt find Domain.unscoped.find_by_name(domain.name) so my theory was that the table is not fully loaded yet?

Reply:
I'm not sure exactly what its issue, since it doesnt reproduce with debug_js

Reply:
This query is on the backend, so not loading the full table shouldn't matter. I wonder if the page displays an error why it couldn't be saved. It would be great if we had more logs. Perhaps something like [capybara-screenshot](https://github.com/mattheworiordan/capybara-screenshot) with some magic on GitHub Actions to archive those could help?

Reply:
This seems like some research, I'm out for a week now :) , just wanted to clean up the test failures so we wont have to check everytime why are they red, and then see its the orgs randomly failing.
The backend issue also happened here: https://github.com/theforeman/foreman/pull/10498